cypress/integration/inventory/moving-items.spec.js

```diff
@@ -9,30 +9,95 @@ import HoldingsRecordView from '../../support/fragments/inventory/holdingsRecord
 import InventoryInstances from '../../support/fragments/inventory/inventoryInstances';
 import InventoryViewSource from '../../support/fragments/inventory/inventoryViewSource';
 import Features from '../../support/dictionary/features';
-import OrdersHelper from '../../support/fragments/orders/ordersHelper';
+import permissions from '../../support/dictionary/permissions';
+import getRandomPostfix from '../../support/utils/stringTools';
 
 const successCalloutMessage = '1 item has been successfully moved.';
-
+let userId = '';
+let firstHolding = '';
+let secondHolding = '';
+const ITEM_BARCODE = `test${getRandomPostfix()}`;
 
 describe('ui-inventory: moving items', () => {
   beforeEach('navigates to Inventory', () => {
-    // TODO: replace this user with all permissions to user with only needed permission for moving items
-    cy.login(Cypress.env('diku_login'), Cypress.env('diku_password'));
-    cy.visit(TopMenu.inventoryPath);
+    cy.createTempUser([
+      permissions.inventoryAll.gui,
+      permissions.uiInventoryMoveItems.gui
+    ])
+      .then(userProperties => {
+        userId = userProperties.userId;
+        cy.login(userProperties.username, userProperties.password);
+        cy.visit(TopMenu.inventoryPath);
+        cy.getToken(Cypress.env('diku_login'), Cypress.env('diku_password'))
+          .then(() => {
+            cy.getLoanTypes({ limit: 1 });
```

Comment:
move complicated data preparing into appropriate fragment

Reply:
It seems excess, because we use separate and understanding methods like getLoanTypes, createInstance etc

Reply:
It seems excess because we use simple and understanding methods likegetLoanTypes, createInstance etc
Also I see a whole promise chain in one place, It gets debugging easilier